### PR TITLE
Release v4.1.1

### PR DESCRIPTION
**Pull request by ghoneycutt**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v4.1.1](https://github.com/voxpupuli/puppet-snmp/tree/v4.1.1) (2019-02-07)
```
Comment (alexjfisher) [inline CHANGELOG.md]: should this be 5.0.0 because we have breaking changes listed?

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,10 +43,6 @@ These should not affect the functionality of the module.
 
 [Full Changelog](https://github.com/voxpupuli/puppet-snmp/compare/3.9.0...v4.0.0)
 
-This is the first release in VoxPupuli's `puppet` namespace.
```
Comment (alexjfisher) [inline CHANGELOG.md]: What happened to this release summary?

Comment (ghoneycutt) [reply CHANGELOG.md]: Dunno.. just ran the github_changelog_generator

File: CHANGELOG.md (hunk 1 above)
Comment (ghoneycutt) [reply CHANGELOG.md]: what breaking changes?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v4.1.1](https://github.com/voxpupuli/puppet-snmp/tree/v4.1.1) (2019-02-07)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-snmp/compare/v4.1.0...v4.1.1)
+
+**Breaking changes:**
+
+- modulesync 2.5.1 and drop Puppet 4 [\#177](https://github.com/voxpupuli/puppet-snmp/pull/177) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (alexjfisher) [inline CHANGELOG.md]: Dropping support for puppet 4 I think is being treated as breaking.
@bastelfreak?

File: CHANGELOG.md (hunk 2 above)
Comment (alexjfisher) [reply CHANGELOG.md]: With the github version from the Gemfile?
Release summaries is quite a new feature and unfortunately there hasn’t been an actual release of gcg for a long time now.

File: CHANGELOG.md (hunk 3 above)
Comment (bastelfreak) [reply CHANGELOG.md]: correct. this is listed as breaking change so it should be a 5.0.0 release.

**Comment by ghoneycutt** [comment]
If we're going to do a major version bump, we should see if there are any other breaking changes that should be merged.

**Comment by ghoneycutt** [comment]
Working on adding data types and deprecating params in #181

**Comment by ghoneycutt** [comment]
Closing as the next release should be v5 and should include PR #181

**Pull request closed without merging**